sdk-wasm-js/src/hash.rs

```diff
@@ -7,7 +7,19 @@ use {
 
 #[wasm_bindgen]
 #[derive(Debug, PartialEq, Eq, Clone)]
-pub struct Hash(pub(crate) solana_hash::Hash);
+pub struct Hash {
+    pub(crate) inner: solana_hash::Hash,
+}
+
+impl Hash {
+    pub fn new(inner: solana_hash::Hash) -> Self {
+        Self { inner }
+    }
+
+    pub fn inner(&self) -> &solana_hash::Hash {
+        &self.inner
+    }
```

Comment:
Think inner is fine, but you could also consider `impl Deref for Hash` so the consumer could call right into the inner type's methods directly.